src/app/legacy/containers/ConsentBanner/index.canonical.jsx

```diff
@@ -15,6 +18,8 @@ const Canonical = ({ onDismissFocusRef }) => {
     handleCookieBannerRejected,
   } = useConsentBanners();
 
+  if (isApp) return null;
```

Comment:
**nitpick:** not sure if this is called out in coding standards anywhere, but it's (my?) preference to always use `{}` after an if statement 
```suggestion
  if (isApp) {
    return null;
  }
```

Reply:
Could be one to discuss and apply a Prettier rule or something if the majority agrees? I kinda like the single-line approach personally, but can see both sides.

Reply:
Feel free to merge this as is @amoore108, we can decide at a later date if we want to change the prettier / lint rules 👍